pkg/provisioner/k8sprovisioner.go

```diff
@@ -110,6 +110,29 @@ func (p k8sProvisioner) newWorkerPod(args WorkerArgs) v1.Pod {
 			MountPath: repoVolumeMountPath,
 		},
 	}
+	gitVolumeMounts := append(volumeMounts, v1.VolumeMount{
+		Name:      "ssh",
```

Comment:
Might be nice to extract into a const since it's required (or is it?) to be the same as the volume name below.

Reply:
Sure